sys/log/full/src/log_shell.c

```diff
@@ -44,6 +47,12 @@ shell_log_dump_entry(struct log *log, struct log_offset *log_offset,
     char data[128];
```

Comment:
This kind of restricts the data length for the cbor entry. You might want to be able to read a longer indefinite length entry.

Reply:
@vrahane this is not a change in this PR. I can add separate patch to this PR with adoption of this log data size - @vrahane have any idea how big it should be ?